leap.js

```diff
@@ -1,3 +1,4 @@
+//my final
```

Comment:
_No no no :rage: !_

You can't just remove everything and write your code..
You have to keep the original file structure to make tests works !

```javascript
'use strict';

/**
 * Represents a year to check whether is leap or not
 *
 * @param  {number} year
 * Numeric year.
 */
function Year(year) {
  this.year = year;
}

/**
 * Whether given year is a leap year.
 *
 * @return {boolean}
 * Whether given year is a leap year.
 */
Year.prototype.isLeap = function () {
//
// YOUR CODE GOES HERE
// (you have to use this.year)
//
}

module.exports = Year;
````
Do you see the little part with the text **_YOUR CODE GOES HERE_** ???
I will give you a little tips: you have to write you code here :wink: !